base/ltspace.dtx

```diff
@@ -568,7 +568,7 @@
 % \changes{LaTeX2e}{1993/12/08}
 %         {Command reimplemented; late birthday present for Chris}
```

Comment:
You were so gracious back then, two decades ago! :-)

Reply:
right ... but I still am ... tell me your wishes for next December ...